lib/graphql_devise/rails/routes.rb

```diff
@@ -35,9 +34,11 @@ def mount_graphql_devise_for(resource, opts = {})
           new_mutation
         end
 
-        GraphqlDevise::Types::MutationType.field("#{mapping_name}_#{action}", mutation: used_mutation)
+        GraphqlDevise::Types::MutationType.field("#{mapping_name}_#{action.to_s.underscore}", mutation: used_mutation)
       end
 
+      Devise.mailer.send(:add_template_helper, GraphqlDevise::MailerHelper)
```

Comment:
There might be an engine way not to need this, but is the method not public?

Reply:
Let me take a look, but it is private [doc](https://apidock.com/rails/AbstractController/Helpers/ClassMethods/add_template_helper)